include/Parser.h

```diff
@@ -43,7 +43,7 @@ class PacketParser {
   void processLineResult(err_code_t& err);
 
 
-  std::queue<struct ::iovec> m_requestKeys;
+  std::queue<struct iovec> m_requestKeys;
```

Comment:
"struct iovec" "struct ::iovec" both exists in source code，leave "struct iovec"。